apps/ausweis/src/features/onboarding/screens/welcome.tsx

```diff
@@ -32,10 +34,16 @@ export default function OnboardingWelcome({ goToNextStep }: OnboardingWelcomePro
           </YStack>
           <YStack flex-1 />
           <XStack gap="$2" my="$6">
-            <Button.Outline p="$0" width="$buttonHeight">
-              <HeroIcons.GlobeAlt size={24} />
+            <Button.Outline
+              p="$0"
+              width="$buttonHeight"
+              onPress={() => setSelectedFlow(selectedFlow === 'c' ? 'bprime' : 'c')}
```

Comment:
should use callback where the current value is passed to prevent weird inconsistencies

```suggestion
              onPress={() => setSelectedFlow((selectedFlow) => selectedFlow === 'c' ? 'bprime' : 'c')}
```